bin/eslint.js

```diff
@@ -139,21 +139,26 @@ runWithTiming("engineConfig", function () {
   }
 });
 
+var cli = runWithTiming("cliInit", function() { return new CLIEngine(options); });
+
 var analysisFiles = runWithTiming("buildFileList", function() {
   return buildFileList(options.extensions);
 });
-var cli = runWithTiming("cliInit", function() { return new CLIEngine(options); }),
-    report = runWithTiming("cliRun", function() { return cli.executeOnFiles(analysisFiles); });
+
+var report = runWithTiming("cliRun", function() { return cli.executeOnFiles(analysisFiles); });
 
 runWithTiming("resultsOutput",
   function() {
     report.results.forEach(function(result) {
       var path = result.filePath.replace(/^\/code\//, "");
-
-      result.messages.forEach(function(message) {
-        var issueJson = buildIssueJson(message, path);
-        console.log(issueJson + "\u0000");
-      });
+      if (cli.isPathIgnored(path)) {
```

Comment:
If we move the check for `isPathIgnored` to `buildFileList`, I don't think we'll see this warning `result` show up, as the path will no longer be passed explicitly to `executeOnFiles`. Since we often see timeout issues with this engine already, I'm hesitant to add another loop on file paths before we hit the analysis stage. What do you think?